src/bin/tikv-dump.rs

```diff
@@ -53,7 +53,7 @@ fn main() {
     opts.optflag("h", "help", "print this help menu");
     opts.optflag("", "info", "print the region info");
     opts.optopt("i", "index", "set the raft log index", "");
-    opts.optopt("k", "key", "set the query raw key, in hex format", "");
+    opts.optopt("k", "key", "set the query raw key, in pretty format", "");
```

Comment:
No, key is in hex format. Otherwise we should change the implementation of `dump_raw_value`.